app/models/event.rb

```diff
@@ -64,12 +66,11 @@ def respondents
   end
 
   def eligible_people
-    Person.active.all # In the future, this will need to honor department
-    # def eligible_people
-    #   self.departments.each do |department|
-    #
-    #   end
-    # end
+    eligible_people = []
```

Comment:
The `has many through` is the other variant of `has and belongs to many`, which I used here. I did not need any additional info on the join table.